shared/actions/notifications.js

```diff
@@ -46,6 +47,16 @@ const setupEngineListeners = () => {
     'keybase.1.NotifyBadges.badgeState',
     ({badgeState}, _: any, getState) => [NotificationsGen.createReceivedBadgeState({badgeState})]
   )
+
+  getEngine().setIncomingActionCreators(
+    'stellar.1.notify.paymentNotification',
+    ({accountID}, _: any, getState) => [
+      // Need to reload the account fully so that assets update
+      WalletsGen.createLoadAccounts(),
```

Comment:
We can keep this short term but longer term I think these things should dispatch a simple single action and the saga should coordinate what side effects should happen